src/item.cpp

```diff
@@ -2093,18 +2093,22 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
 
 std::string item::display_name(unsigned int quantity) const
 {
+    std::string tn = tname(quantity);
+
+    if (get_side() != BOTH) {
+        tn += string_format(" (%s)", get_side() == LEFT ? _("left")  : _("right"));
```

Comment:
Instead of doing multiple different append operations, it's easier to understand the code if you set a "laterality" string variable here that's either " (left)", " (right)", or "" here, and apply it to the format string on one of the later lines.